src/pages/patientView/clinicalInformation/ClinicalInformationPatientTable.tsx

```diff
@@ -33,14 +34,15 @@ export default class ClinicalInformationPatientTable extends React.Component<
         value: string;
     }): string {
         let ret: string;
-        switch (data.attribute) {
-            case 'Overall Survival (Months)':
-                ret = parseInt(data.value, 10).toFixed(0);
-                break;
-            default:
-                ret = data.value;
-                break;
+
+        const parsedFloat = parseFloat(data.value);
```

Comment:
Corey, this works but really we should just be testing for isNumeric first and then proceeding.   you can use lodash 
_.isNumeric function to do this.

Reply:
I'm seeing that function as being deprecated. I can create a new utility function for this which is basically isNumeric https://github.com/lodash/lodash/issues/1148#issue-69568538
``` Javascript 
function isNumeric(x: any) {
     return !(isNaN(x)) && (typeof x !== "object") &&
       (x != Number.POSITIVE_INFINITY) && (x != Number.NEGATIVE_INFINITY);
}

Where should this function go?

Reply:
Maybe we use [isFinite](https://lodash.com/docs/4.17.15#isFinite) instead?

Reply:
After testing, `isFinite` does not consider any string (even a number in a string like `const x = '2'`) as true. Which doesn't solve the problem Aaron mentioned of testing before converting `data.value`, because we would have to convert the string to see if `isFinite` would be true.